aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -742,6 +744,28 @@ Tensor& fmod_(Tensor& self, Scalar other) {
   return at::fmod_out(self, self, other);
 }
 
+Tensor& logaddexp_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  auto iter = TensorIterator::binary_op(result, self, other);
```

Comment:
Do you need a `/*check_mem_overlap=*/=true` here? What if result is self or other?

Reply:
there will be no problem if result is self or other, but problem might occur when view of tensor overlap with other view, so `check_mem_overlap` is definitely needed here.